.github/workflows/tests.yml

```diff
@@ -9,12 +9,13 @@ jobs:
 
   tests:
     name: Test code
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
     strategy:
       matrix:
         go:
           - ^1.18
-          - ^1.19
+          # FIXME https://github.com/grafana/carbonapi/issues/70
+          # - ^1.19
```

Comment:
Can be removed. Issue was fixed

Reply:
I can remove that later myself, just not to block the PR